extensions/machine-learning/src/views/wizardView.ts

```diff
@@ -112,12 +112,6 @@ export class WizardView extends MainViewBase {
 	}
 
 	public async refresh(): Promise<void> {
-		for (let index = 0; index < this._pages.length; index++) {
```

Comment:
when does the pages get added - once and keep that way?

Reply:
pages are added when wizard is created, I added this code so if after wizard is created I can add new pages. It's causing issues and I don't need it in any wizard right now 

---
In reply to: [592657209](https://github.com/microsoft/azuredatastudio/pull/14659#discussion_r592657209) [](ancestors = 592657209)